CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ else()
     MESSAGE("  OpenBLAS_INCLUDE: ${OpenBLAS_INCLUDE_DIR}")
 endif()
 
-find_package( OpenCV 3.3 REQUIRED COMPONENTS core imgproc calib3d highgui objdetect)
+find_package( OpenCV 3.4 REQUIRED COMPONENTS core imgproc calib3d highgui objdetect)
```

Comment:
In develop the CMakeList.txt should not need to change.

Reply:
I thought that would be the case. For some reason it was throwing dependency errors before, and I had to change it to match the compiled version of opencv. The new PR will have 3.3 here.